policy.tex

```diff
@@ -1,67 +1,67 @@
 \section{Release Policy} \label{sec:policy}
 
-The following policies are provided taking into account the requirements described in the previous section \ref{sec:reqdef}.
+The following policies are derived from the consolidated requirements described in the previous section \ref{sec:reqdef}.
 
 
 \subsection{Versioning Policy} \label{sec:versinopolicy}
 
 The DM release versioning shall follow Semantic Versioning\footnote{\url{https://semver.org/}} as described in \citeds{DMTN-106}, section 3.3.
 
-This policy partially answers to \ref{sec:stable} requirement.
-In general, API stability is a development aspect that needs to be addressed in the \citep{DevGuide}.
+This policy partially addresses the requirement for stable APIs (\S\ref{sec:stable}).
+
+% Removed this comment, because it is not release policy.
+%In general, API stability is a development aspect that needs to be addressed in the \citep{DevGuide}.
 
 
 \subsection{Release Schedule Policy} \label{sec:schedulepolicy}
 
 Major and minor releases should be scheduled accordingly to the requirements of each stakeholder.
 
-A release plan including for each release following information:
+Each release shall be made following a release plan, which provides the following information:
 
 \begin{itemize}
-\item when a release is expected
-\item specify the corresponding milestone or RFC
-\item if it is a time-based release: specify the timing when the release shall be done (for example, every 6 months)
-\item if it is a functionality based release: specify which functionalities shall be included in the release
+\item when the release is expected;
+\item the specify the corresponding milestone or RFC;
```

Comment:
'the specify the corresponding...' doesn't sound correct to me... but I may be wrong.

Reply:
It's not. Good catch!